CMakeLists.txt

```diff
@@ -264,14 +257,22 @@ if (FMT_INSTALL)
               "Installation directory for pkgconfig (.pc) files, a relative path "
               "that will be joined to ${CMAKE_INSTALL_PREFIX}, or an arbitrary absolute path.")
 
+  get_filename_component(FMT_INC_DIR_PARENT "${FMT_INC_DIR}" DIRECTORY)
```

Comment:
I think we should remove `/fmt` from `FMT_INC_DIR` for consistency with `FMT_LIB_DIR` and `CMAKE_INSTALL_INCLUDEDIR`.